src/scss/abstracts/_variables.scss

```diff
@@ -23,6 +23,7 @@ $phovea-theme-colors: (
 $theme-colors: map-merge($theme-colors, $phovea-theme-colors);
 
 $phovea-loading-icon-url: url('~phovea_ui/dist/assets/caleydo_c_anim.svg') !default;
+$phovea-loading-icon-size: 10% !default;
```

Comment:
The 10% seems to cause the a flickering in size when the view is loading. 

![caleydo-logo-size](https://user-images.githubusercontent.com/5851088/127784016-7609952a-62c2-461c-9b96-8de4027fbe7f.gif)

I guess that the percentage size is dependening on the size of a parent element and once the size changes the size of the background changes as well.

Maybe using `auto` (= browser default value) for the [`background-size`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size) is the better option here?

```suggestion
$phovea-loading-icon-size: auto !default;
```

Can you please have a look again?

Reply:
I see you have already committed the fix. It works in my tests, thank you! :+1: